packages/cli/src/metadataGeneration/methodGenerator.ts

```diff
@@ -139,19 +139,9 @@ export class MethodGenerator {
     this.produces = this.getProduces();
   }
 
-  private getProduces(): string | undefined {
-    const producesDecorators = this.getDecoratorsByIdentifier(this.node, 'Produces');
-
-    if (!producesDecorators || !producesDecorators.length) {
-      return;
-    }
-    if (producesDecorators.length > 1) {
```

Comment:
ok @WoH so here is the deal: currently we allow only one `@Produces`. Also it's possible to put only one media type to `SuccessResponse` and `Response` - however according to swagger specification docs it's possible to specify multiple media types for given response.